crates/interpreter/src/lib.rs

```diff
@@ -7,23 +11,22 @@ mod macros;
 
 pub mod gas;
 mod host;
-pub mod inner_models;
-pub mod instruction_result;
+mod inner_models;
+mod instruction_result;
 pub mod instructions;
 mod interpreter;
 
 pub(crate) const USE_GAS: bool = !cfg!(feature = "no_gas_measuring");
 
 // Reexport primary types.
 pub use gas::Gas;
-pub use host::{DummyHost, Host};
+pub use host::*;
```

Comment:
Intend to move all the things to name inclusion, `*` hides what is inside and was good for developments but specified names give better clarity to what is exported.